docs/user/README.md

```diff
@@ -0,0 +1,63 @@
+
+The Telemetry module focuses exactly on these aspects (instrumentation/collection/shipment) happening in the runtime and explicitly defocuses on backends. In addition, you can check the [tutorials](05-tutorials.md) on how to install lightweight in-cluster backends for demo or development purposes.
+
+## Features
+
+The Telemetry module enables your application with the telemetry support by providing the following:
+
+- guidance for the instrumentation - based on [Open Telemetry](https://opentelemetry.io/), community samples are provided on how to instrument your code using the [Open Telemetry SDKs](https://opentelemetry.io/docs/instrumentation/) in nearly any programming language
+- tooling for collection, filtering, and shipment - based on the [Open Telemetry Collector](https://opentelemetry.io/docs/collector/), you can configure basic pipelines to filter and ship telemetry data
+- integration in a vendor-neutral way to a vendor-specific observability system - based on the [OpenTelemetry protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/), backend systems can be integrated
+- opt-out from features for advanced scenarios - at any time you can opt out per data type and execute the telemetry data collection/shipment with custom tooling
+- SAP BTP as first-class integration - integration into BTP Observability services will be prioritized
+- Enterprise-grade qualities - the setup is battle-tested and will satisfy typical development standards
+
+## Scope
+
+There will be a focus on the signals of application logs, distributed traces, and metrics only. Other kinds of signals are not considered. Also, logs like audit logs or operational logs are specifically descoped.
+
+Supported integration scenarios are neutral to the vendor of the target system; a vendor-specific way is not planned at the moment.
+
+Initially,
+- logs will not be based on the vendor-neutral OTLP protocol; follow this [epic](https://github.com/kyma-project/kyma/issues/16307) to understand the current progress on that
+
+
+## Components
+
+![Components](./assets/general-components.drawio.svg)
+
+### Telemetry Operator
+
+The module ships the Telemetry Operator as its heart component. The operator implements the Kubernetes controller pattern and manages the whole lifecycle of all other components covered in the module. The operator watches for LogPipeline, TracePipeline, and MetricPipeline resources created by the user. With these, the user describes in a declarative way what data of a signal type to collect and where to ship it.
```

Comment:
```suggestion
The module ships the Telemetry Operator as its core component. The operator implements the Kubernetes controller pattern and manages the whole lifecycle of all other components covered in the module. The operator watches for LogPipeline, TracePipeline, and MetricPipeline resources created by the user. With these, the user describes in a declarative way what data of a signal type to collect and where to ship it.
```

This paragraph appears in another document as well, right? Make sure all the changes applied to one document are also apply to the other doc.